src/styles/StyleUtils.ts

```diff
@@ -201,24 +181,17 @@ function getAvatarBorderStyle(size, type) {
 
 /**
  * Helper method to return old dot default avatar associated with login
- *
- * @param {String} [workspaceName]
- * @returns {Object}
  */
-function getDefaultWorkspaceAvatarColor(workspaceName) {
+function getDefaultWorkspaceAvatarColor(workspaceName: string): ViewStyle | CSSProperties {
     const colorHash = UserUtils.hashText(workspaceName.trim(), workspaceColorOptions.length);
 
     return workspaceColorOptions[colorHash];
 }
 
 /**
  * Takes safe area insets and returns padding to use for a View
- *
- * @param {Object} insets
- * @param {Number} [insetsPercentage] - Percentage of the insets to use for sides and bottom padding
- * @returns {Object}
  */
-function getSafeAreaPadding(insets, insetsPercentage = variables.safeInsertPercentage) {
+function getSafeAreaPadding(insets: EdgeInsets, insetsPercentage: number = variables.safeInsertPercentage): ViewStyle | CSSProperties {
```

Comment:
```suggestion
function getSafeAreaPadding(insets: EdgeInsets | undefined, insetsPercentage: number = variables.safeInsertPercentage): ViewStyle | CSSProperties {
```

Consider that `EdgeInsets` may be `undefined`, for some places that use it don't have `EdgeInsets` defined always.

Reply:
Done!